app/src/main/kotlin/at/bitfire/davdroid/repository/AccountRepository.kt

```diff
@@ -165,8 +166,17 @@ class AccountRepository @Inject constructor(
     fun fromName(accountName: String) =
         Account(accountName, accountType)
 
+    /**
+     * Gets all real accounts (but not address book accounts)
```

Comment:
_Address book accounts_ shouldn't be mentioned anywhere in this class except in the class KDoc to say that this class is not about _address book accounts_ (already there).